### PR TITLE
Feature/faq school

### PR DESCRIPTION
**Pull request by avkamenev**

### CODE REVIEW

File: deeppavlov/models/faq/faq_cos.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+"""
```
Comment (nikolay-bushkov) [inline deeppavlov/models/faq/faq_cos.py]: Use # to comment license information.

File: deeppavlov/configs/faq/fasttext_avg_autofaq.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        "out": "q_token_lemmas"
+      },
+      {
+        "name": "fasttext",
+        "in": "q_token_lemmas",
+        "load_path": "embeddings/lenta_lower_100.bin",
+        "out": "tokens_fasttext_vectors"
+      },
+      {
+        "name": "sentence2vector_v2w_avg",
```
Comment (nikolay-bushkov) [inline deeppavlov/configs/faq/fasttext_avg_autofaq.json]: v2w or w2v?

File: deeppavlov/models/faq/faq_logreg.py [hunk 3]
```diff
@@ -40,6 +54,14 @@ def __init__(self, c=1, penalty='l2', save_path: str = None, load_path: str = No
             self.load()
 
     def __call__(self, q_vect) -> Tuple[List[str], List[str]]:
```
Comment (nikolay-bushkov) [inline deeppavlov/models/faq/faq_logreg.py]: Type of input parameters? Components should accept batches.

File: deeppavlov/models/faq/faq_cos.py [hunk 4]
```diff
@@ -68,11 +99,15 @@ def fit(self, x_train_vects, y_train) -> None:
         self.y_train = list(y_train)
 
     def save(self) -> None:
+        """Save FAQ model
```
Comment (nikolay-bushkov) [inline deeppavlov/models/faq/faq_cos.py]: Looks ugly, isn't it? Try one line... Also, avoid obvious descriptions.

File: deeppavlov/models/faq/faq_cos.py [hunk 5]
```diff
@@ -31,14 +33,34 @@
 
 @register("faq_cos_model")
 class FaqCosineSimilarityModel(Estimator, Serializable):
+    """
+    FAQ model based on cosine similarity between vectorized sentences
+    """
 
     def __init__(self, save_path: str = None, load_path: str = None, **kwargs) -> None:
+        """FAQ model based on cosine similarity between vectorized sentences
```
Comment (nikolay-bushkov) [inline deeppavlov/models/faq/faq_cos.py]: We agreed to put __init__() docstrings in class docstrings.

File: docs/apiref/models/vectorizers.rst [hunk 6]
```diff
@@ -1,10 +1,17 @@
 deeppavlov.models.vectorizers
 =============================
 
-.. automodule:: deeppavlov.models.vectorizers.hashing_tfidf_vectorizer
```
Comment (nikolay-bushkov) [inline docs/apiref/models/vectorizers.rst]: Why another TFIDF vectorizer?

File: deeppavlov/models/faq/faq_cos.py [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+        Returns:
+            None
+        """
+        self.save_path = save_path
+        self.load_path = load_path
+        if kwargs['mode'] != 'train':
+            self.load()
+
+    def __call__(self, q_vect) -> Tuple[List[str], List[str]]:
```
Comment (nikolay-bushkov) [inline deeppavlov/models/faq/faq_cos.py]: Will np.round(cos_similarities[answer_id], 2) return string?

Comment (avkamenev) [reply deeppavlov/models/faq/faq_cos.py]: np.round(cos_similarities[answer_id], 2) returns "score"

File: deeppavlov/models/faq/faq_cos.py (hunk 4 above)
Comment (avkamenev) [reply deeppavlov/models/faq/faq_cos.py]: done.

**Pull request merged**